\documentclass[10pt]{article}
\usepackage{refcount}
\usepackage{charter}
\usepackage{fullpage}
\usepackage[T1]{fontenc}
\usepackage[resetlabels]{multibib}
\usepackage[ManyBibs,LabelsAligned]{currvita}
\usepackage{ifpdf}
\usepackage{denselists}

% \usepackage[defernumbers=true,sorting=reverse,backend=biber]{biblatex}

\ifpdf
  \usepackage[pdftex,pdfpagelabels=false]{hyperref}
\else
  \usepackage[hypertex,pdfpagelabels=false]{hyperref}
\fi
\usepackage{setspace}
\hypersetup{
  pdfauthor={Jason E Stajich}
  pdftitle={CV for Jason Stajich}
  pdfsubject={Detailed CV for Jason},
  urlcolor=blue,
}

\usepackage{charter}

% Initialize each paper type for which you need a bibliography.
\newcites{journal}{Publications}
\newcites{genome}{Genome Reports}
\newcites{submitted}{Submitted Manuscripts and Preprints}
\newcites{book}{Books and Book Chapters}
\newcites{review}{Reviews (Refereed)}
\newcites{consortia}{Consortia publications}
\newcites{report}{Meeting and Technical Reports}
\newcites{essay}{Essays}
\newcites{commentary}{Commentaries and Book Reviews}

\makeatletter
\long\def\thebibliography#1{%
  \section*{\refname}%
  \@mkboth{\MakeUppercase\refname}{\MakeUppercase\refname}
  \settowidth{\dimen0}{\@biblabel{#1}}%
  \setlength{\dimen2}{\dimen0}%
  \addtolength{\dimen2}{\labelsep}
  \sloppy
  \clubpenalty 4000
  \@clubpenalty
  \clubpenalty
  \widowpenalty 4000
  \sfcode `\.\@m
  \renewcommand{\labelenumi}{\@biblabel{\theenumi}} % labels like [3], [2], [1]
  \begin{etaremune}[labelwidth=\dimen0,leftmargin=\dimen2]\@openbib@code
}
\def\endthebibliography{\end{etaremune}}
\def\@bibitem#1{%
  \item \if@filesw\immediate\write\@auxout{\string\bibcite{#1}{\the\value{enumi}}}\fi\ignorespaces
}
\makeatother

% Specify the bibtex style that you want for each paper type
\bibliographystyle{nih}
\bibliographystylejournal{nih}
\bibliographystylesubmitted{nih}
\bibliographystylegenome{nih}
\bibliographystylereview{nih}
\bibliographystylebook{nih}
\bibliographystylereport{nih}
\bibliographystylecommentary{nih}
\bibliographystyleessay{nih}
\bibliographystyleconsortia{nih}

\newlength{\oldcvlabelwidth}
\renewcommand*{\cvbibname}{}

\begin{document}

% This is what will appear at the very top of your CV.
\begin{cv}{\centerline{Jason E. Stajich}}

% Items have more vertical space between them than line breaks.
% Note: if you want a linebreak after each cvlist title, you can do:
% \begin{cvlist}{Title \hspace{3in}}

\setlength{\cvlabelwidth}{18mm}
\begin{cvlistcompact}{Contact}
\item{Department of Microbiology \& Plant Pathology and Institute for
  Integrative Genome Biology, \\
  University of California, Riverside, CA 92521}
  \item{ \textbf{Telephone:} +1 951-827-2363 \textbf{Email:}
    jason.stajich@ucr.edu \\
    \textbf{Web:} \url{http://lab.stajich.org/}}
\end{cvlistcompact}

\begin{cvlistcompact}{\bf Education}
\item [2006--2009] Postdoctoral training, University of California, Berkeley, CA. Mentor: Dr. John W Taylor
\item [2001--2006] Ph.D., Genetics and Genomics, Duke University, Durham, NC. Advisor: Dr. Fred S Dietrich
\item [1995--1999] B.S., Computer Science, Duke University, Durham, NC
\end{cvlistcompact}

\begin{cvlistcompact}{\bf Academic appointments:}
\item [2017--] Professor (Step VII), Dept of Microbiology \& Plant Pathology. University of California Riverside.
\item [2014--2017] Associate Professor (with tenure), Dept of Microbiology \& Plant Pathology. University of California Riverside.
\item [2009--2014] Assistant Professor, Dept of Plant Pathology \& Microbiology. University of California Riverside.
\item [2006--2009] Postdoctoral Research Fellow. Miller Institute for Basic Research. \\
  Dept of Plant and Microbial Biology, University of California
  Berkeley. \\
\end{cvlistcompact}

\begin{cvlistcompact}{Honors and Awards:}
\item [2020] Fellow, American Association for the Advancement of Science
\item [2020] Fellow, Mycological Society of America
\item [2020] Fellow, American Academy of Microbiology, American
  Society for Microbiology
\item [2019--2025] CIFAR Fellow in program 'Fungal Kingdom: Threats \& Opportunities'
\item [2019] Rosie Perez Memorial Seminar, North Carolina State University
\item [2017] Whetzel-Westcott-Dimock Special Lecturer, Cornell University
\item [2015] Kavli Fellow, Kavli Frontiers of Science
\item [2014] C. J. Alexopoulos Prize, Mycological Society of America
\item [2006--2009] Miller Institute for Basic Research in Science,
  Postdoctoral Research Fellowship
\item [2003--2006] National Science Foundation, Graduate Research Fellowship
\end{cvlistcompact}

% These commands will make our papers be indented after the first line
% to make the author's name stand out more.  Also, we reduce the
% indentation of the paper type titles (e.g., Journal Papers,
% Conference Paper) compared to regular cvlist items.
\setlength{\oldcvlabelwidth}{\cvlabelwidth}
\setlength{\cvlabelwidth}{1em}
\renewcommand*{\bibindent}{1.5em}
% Do bibaddtoleftmargin instead of biblabelsep if you're
% not using manybib Or open bib
%\renewcommand*{\bibaddtoleftmargin}{1.5em}
\renewcommand*{\biblabelsep}{1.5em}

% To update, before doing latexmk, run:
% bibtex journal
% bibtex refpaper
% bibtex unrefpaper
% bibtex techreport

\makeatletter
\def\thebibliography#1{\list
{\arabic{enumiv}.}{\settowidth\labelwidth{[#1]}\leftmargin\labelwidth
\advance\leftmargin\labelsep
\usecounter{enumiv}}
\def\newblock{\hskip .11em plus .33em minus .07em}
\sloppy\clubpenalty4000\widowpenalty4000
\sfcode`\.=1000\relax
}
\let\endthebibliography=\endlist
\makeatother

\begin{cvlist}{Publications:}
\item \textbf{Peer Reviewed Publications}
  \def\mybibitem#1{\item}
  \nocitejournal{GluckThaler2025}
  \nocitejournal{Shyong2025}
  \nocitejournal{Santos2024}
  \nocitejournal{AlmeidaPaes2024}
  \nocitejournal{Yugueros2024}
  \nocitejournal{Carpenter2024}
  \nocitejournal{Chander2024}
  \nocitejournal{Eulalio2024}
  \nocitejournal{VargasGastelum2024}
  \nocitejournal{Tenorio2024}
  \nocitejournal{Coleine2024b}
  \nocitejournal{Stajich2024}
  \nocitejournal{Clemons2024}
  \nocitejournal{Lovett2024}
  \nocitejournal{Coleine2024a}
  \nocitejournal{Kirkland2023}
  \nocitejournal{Lin2023}
  \nocitejournal{Ettinger2023c}
  \nocitejournal{Adams2023}
  \nocitejournal{Leo2023}
  \nocitejournal{Ettinger2023b}
  \nocitejournal{Gryganskyi2023}
  \nocitejournal{Murante2023}
  \nocitejournal{Meili2023}
  \nocitejournal{Merenyi2023}
  \nocitejournal{Kurbessoian2023b}
  \nocitejournal{Hansen2023}
  \nocitejournal{Hanafy2023}
  \nocitejournal{Yuan2023}
  \nocitejournal{Ettinger2023a}
  \nocitejournal{Wang2023}
  \nocitejournal{Xi2023}
  \nocitejournal{Chander2023}
  \nocitejournal{Liu2023}
  \nocitejournal{Reynolds2023}
  \nocitejournal{Lofgren2022b}
  \nocitejournal{Winter2022}
  \nocitejournal{Ettinger2022c}
  \nocitejournal{Yuan2022}
  \nocitejournal{NarunskyHaziza2022}
  \nocitejournal{MeloTeixeira2022a}
  \nocitejournal{Amses2022}
  \nocitejournal{Gryganskyi2022}
  \nocitejournal{Chang2022}
  \nocitejournal{Maccaro2022}
  \nocitejournal{Blachowicz2022}
  \nocitejournal{Maltz2022}
  \nocitejournal{MalarC2022}
  \nocitejournal{SouzaPacheco2022}
  \nocitejournal{Lofgren2022a}
  \nocitejournal{Coleine2022b}
  \nocitejournal{MeloTeixeira2022}
  \nocitejournal{Duttke2022}
  \nocitejournal{Parker2022}
  \nocitejournal{Kirkland2021}
  \nocitejournal{Reynolds2021}
  \nocitejournal{Gostincar2021}
  \nocitejournal{Hopke2021}
  \nocitejournal{Ross2021}
  \nocitejournal{Jones2021}
  \nocitejournal{Ettinger2021}
  \nocitejournal{Nielsen2021}
  \nocitejournal{Carlin2021}
  \nocitejournal{MeloTeixeira2021}
  \nocitejournal{Rajewski2021}
  \nocitejournal{Selbmann2021}
  \nocitejournal{Li2021}
  \nocitejournal{Tesei2021}
  \nocitejournal{Demers2021}
  \nocitejournal{Chang2021}
  \nocitejournal{MalarC2021}
  \nocitejournal{Cisse2021}
  \nocitejournal{Kowalski2021}
  \nocitejournal{Jones2021}
  \nocitejournal{Hollin2021}
  \nocitejournal{Albanese2021}
  \nocitejournal{Geiser2020}
  \nocitejournal{ODonnell2020}
  \nocitejournal{Ulzurrun2021}
  \nocitejournal{Lee2021}
  \nocitejournal{EudesFilho2020}
  \nocitejournal{Chen2020}
  \nocitejournal{CarrilloAJ2020}
  \nocitejournal{Vandepol2020}
  \nocitejournal{Collins2020}
  \nocitejournal{Myers2020}
  \nocitejournal{Pombubpa2020}
  \nocitejournal{Tabima2020}
  \nocitejournal{Coleine2020g}
  \nocitejournal{Yang2020}
  \nocitejournal{Macias2020}
  \nocitejournal{Coleine2020f}
  \nocitejournal{Coleine2020e}
  \nocitejournal{Coleine2020c}
  \nocitejournal{Coleine2020b}
  \nocitejournal{Knudsen2020}
  \nocitejournal{Carrillo2020}
  \nocitejournal{Odebode2020}
  \nocitejournal{Uehling2019}
  \nocitejournal{Carrillo2019a}
  \nocitejournal{Kowalski2019}
  \nocitejournal{Wang2019}
  \nocitejournal{Coleine2019b}
  \nocitejournal{Macias2019a}
  \nocitejournal{Murphy2019}
  \nocitejournal{Boyce2019}
  \nocitejournal{Blachowicz2019}
  \nocitejournal{Cisse2019}
  \nocitejournal{Bewick2019}
  \nocitejournal{Chen2019}
  \nocitejournal{Davis2019}
  \nocitejournal{Romsdahl2019}
  \nocitejournal{Chang2018}
  \nocitejournal{Demers2018}
  \nocitejournal{Arvidson2018}
  \nocitejournal{Arnesen2018}
  \nocitejournal{Romsdahl2018a}
  \nocitejournal{Song2018}
  \nocitejournal{Coleine2018b}
  \nocitejournal{Coleine2018a}
  \nocitejournal{Gostincar2018}
  \nocitejournal{Cisse2018}
  \nocitejournal{Wang2018a}
  \nocitejournal{Collins2018}
  \nocitejournal{Gryganskyi2018}
  \nocitejournal{Kirkland2018}
  \nocitejournal{Leonard2018}
  \nocitejournal{Na2018}
  \nocitejournal{Beaudet2017}
  \nocitejournal{Lu2017}
  \nocitejournal{Aryal2017}
  \nocitejournal{Park2017}
  \nocitejournal{Torres-Cruz2017}
  \nocitejournal{Sinha2017}
  \nocitejournal{Mondo2017}
  \nocitejournal{Ahrendt2017}
  \nocitejournal{Nguyen2017}
  \nocitejournal{Chen2017}
  \nocitejournal{Spatafora2016}
  \nocitejournal{Short2017}
  \nocitejournal{Castanera2016}
  \nocitejournal{Lee2016}
  \nocitejournal{deMan2016}
  \nocitejournal{URen2016}
  \nocitejournal{Pieuchot2015}
  \nocitejournal{Wang2015}
  \nocitejournal{Liu2015}
  \nocitejournal{Willger2015}
  \nocitejournal{Collins2015}
  \nocitejournal{Melida2015}
  \nocitejournal{Teixeira2014}
  \nocitejournal{Ouyang2014}
  \nocitejournal{Inglis2014}
  \nocitejournal{Treseder2014}
  \nocitejournal{Janbon2014}
  \nocitejournal{Sachs2014}
  \nocitejournal{Traeger2013}
  \nocitejournal{Gryganskyi2013}
  \nocitejournal{Gioti2013b}
  \nocitejournal{James2013}
  \nocitejournal{Rosenblum2013}
  \nocitejournal{Robb2013}
  \nocitejournal{Jamieson2013}
  \nocitejournal{Cheng2013}
  \nocitejournal{Gioti2013a}
  \nocitejournal{Nygren2012}
  \nocitejournal{Abramyan2012}
  \nocitejournal{Gioti2012}
  \nocitejournal{Stajich2012}
  \nocitejournal{Joneson2011}
  \nocitejournal{Ellison2011}
  \nocitejournal{DSouza2011}
  \nocitejournal{Burns2010}
  \nocitejournal{Smith2010}
  \nocitejournal{Levesque2010}
  \nocitejournal{Strandberg2010}
  \nocitejournal{Ohm2010}
  \nocitejournal{Stajich2010}
  \nocitejournal{Neafsey2010}
  \nocitejournal{Nowrousian2010}
  \nocitejournal{Sharpton2009}
  \nocitejournal{Fisher2009}
  \nocitejournal{Rosenblum2008}
  \nocitejournal{Regier2008}
  \nocitejournal{Martin2008}
  \nocitejournal{Lilly2008}
  \nocitejournal{Hu2008}
  \nocitejournal{Stajich2007a}
  \nocitejournal{Fraser2007}
  \nocitejournal{Harrison2007}
  \nocitejournal{Erwin2007}
  \nocitejournal{Fitzpatrick2006}
  \nocitejournal{Demuth2006}
  \nocitejournal{James2006}
  \nocitejournal{Kaemper2006}
  \nocitejournal{Stajich2006a}
  \nocitejournal{Giles2006}
  \nocitejournal{Cramer2006}
  \nocitejournal{Hesselberth2006}
  \nocitejournal{Stajich2005}
  \nocitejournal{Mitreva2005}
  \nocitejournal{Leman2005}
  \nocitejournal{Hahn2005}
  \nocitejournal{Fraser2005}
  \nocitejournal{Kraus2004}
  \nocitejournal{Stein2003}
  \nocitejournal{Hahn2003}
  \nocitejournal{Stein2002}
  \nocitejournal{Stajich2002}
  \nocitejournal{Baxter2002}
  \bibliographyjournal{stajichpubs}

\item \textbf{Microbial Resource Announcements}
  \nocitegenome{Petronek2024}
  \nocitegenome{Nguyen2024b}
  \nocitegenome{Nguyen2024a}
  \nocitegenome{Kurbessoian2024}
  \nocitegenome{Shea2023}
  \nocitegenome{Kurbessoian2023a}
  \nocitegenome{Stajich2022}
  \nocitegenome{Ettinger2022b}
  \nocitegenome{Ettinger2022a}
  \nocitegenome{Coleine2021}
  \nocitegenome{Ward2021}
  \nocitegenome{Stajich2021}
  \nocitegenome{Coleine2020a}
  \nocitegenome{Coleine2020d}
  \nocitegenome{Teixeira2019}
  \nocitegenome{Coleine2019a}
  \nocitegenome{Kasson2019}
  \nocitegenome{Coleine2017}
  \bibliographygenome{stajichpubs}

\item \textbf{Submitted Manuscripts and Preprints}
  \nocitesubmitted{Moses2025}
  \nocitesubmitted{Mohseni2025}
  \nocitesubmitted{Kelly2025}	
  \nocitesubmitted{Teixeira2025}
  \nocitesubmitted{Ettinger2025b}
  \nocitesubmitted{Ettinger2025a}
  \nocitesubmitted{Long2025}
  \nocitesubmitted{Ettinger2024b}
  \nocitesubmitted{Kelly2024}
  \nocitesubmitted{Kurbessoian2024a}
  \nocitesubmitted{Ettinger2024a}
  \nocitesubmitted{He2023}
  \nocitesubmitted{Yacoub2024}
  \nocitesubmitted{Myers2024}
  \nocitesubmitted{CarterHouse2020}
  \nocitesubmitted{Alvarado2020}

%  \nocitesubmitted{Unruh2019}
  \bibliographysubmitted{stajichpubs}

\item \textbf{Reviews (Refereed)}
  \nocitereview{Case2025}
  \nocitereview{Gostincar2023}
  \nocitereview{Coleine2022a}
  \nocitereview{Lofgren2021}
  \nocitereview{Selbmann2020}
  \nocitereview{Coleine2020h}
  \nocitereview{James2020}
  \nocitereview{Lovett2020}
  \nocitereview{Fisher2020}
  \nocitereview{Warren2019}
  \nocitereview{Amend2019}
  \nocitereview{Hibbett2013}
  \nocitereview{Rosenblum2010}
  \nocitereview{Rosenblum2010a}
  \nocitereview{Stajich2009a}
  \nocitereview{Stajich2006}
  \bibliographyreview{stajichpubs}

\item \textbf{Books and Book Chapters}
  \nocitebook{Gostincar2024}
  \nocitebook{Stajich2017}
  \nocitebook{Spatafora2017a}
  \nocitebook{Stajich2015}
  \nocitebook{Stajich2013}
  \nocitebook{Fisher2012}
  \nocitebook{McKay2010}
  \nocitebook{Edwards2009}
  \nocitebook{Stajich2007}
  \nocitebook{Stajich2006b}
  \nocitebook{Coghlan2006}

  \bibliographybook{stajichpubs}

\item \textbf{Consortia publications}
\nociteconsortia{West2022}
\nociteconsortia{Bidartondo2008}

\bibliographyconsortia{stajichpubs}

\item \textbf{Meeting and Technical Reports}
  \nocitereport{Case2022a}
  \nocitereport{Case2022b}
  \nocitereport{DunningHotopp2020}
  \nocitereport{Pombubpa2020a}
  \nocitereport{Baltrus2019}
  \nocitereport{Gaya2018}
  \nocitereport{Momany2015}
  \nocitereport{Kennedy2015}
  \nocitereport{Glass2014}
  \nocitereport{Bates2013}
  \nocitereport{Lapp2007}

  \bibliographyreport{stajichpubs}

\item \textbf{Commentaries and Book Reviews}
\nocitecommentary{Stajich2024a}
\nocitecommentary{Valent2019}
\nocitecommentary{Stajich2016}
\nocitecommentary{Stajich2011}
\nocitecommentary{Stajich2009b}

\bibliographycommentary{stajichpubs}

\item \textbf{Essays}
  \nociteessay{Stajich2014}
  \bibliographyessay{stajichpubs}

\begin{cvlistcompact}{\bf Software and other Products}
  \item BioPerl - \url{http://bioperl.org} - Core developer
\item Github \url{http://github.com/hyphaltip} - individual
  projects
\item Github \url{http://github.com/stajichlab} - lab
  projects
 \item Protocols.io Protocols \url{https://www.protocols.io/researchers/jason-stajich} - public protocols
 \item Github \url{http://github.com/1KFG} - 1000 Fungal
   genomes project
\item Github \url{http://github.com/zygolife} - ZyGoLife NSF project
  and associated phylogenomics
\item Website: \url{http://1000.fungalgenomes.org} - 1KFG project
\item Website: \url{http://herptilemicrobiomes.org} - NSF URoL Herptile Microbiomes
\item Website: \url{http://zygolife.org} - NSF Zygolife
\item Website: \url{http://dynamiterice.org} - NSF Rice Transposable
    Element project
\item Website: \url{http://fungalgenomes.org/blog} - ``The Hyphal
  Tip'' A Blog I write about Fungal Genomics
\item Website \& Database (Collaboration): \url{http://fungidb.org}
\end{cvlistcompact}


\end{cvlist}
% Reset the label indentation
\setlength{\cvlabelwidth}{24mm}

%\begin{cvlist}{Employment}
%	\item[Summer 2002] Visiting Researcher, Lincoln D Stein laboratory \\
%          Cold Spring Harbor Laboratories, Cold Spring Harbor, NY
%	\item[1999--2001] Analyst/Programmer, Center for Human
%          Genetics \\
%          Duke University Medical Center, Durham, NC
%\end{cvlist}

\begin{cvlistcompact}{\bf Grant Support:}
\item [{\bf Ongoing support}]

\item [2017-2026] National Institutes of Health. R01-AI127548 \\
``Evolved Heterogeneity contributes to chronic fungal lung infections'' \\
Role: Senior Personnel. PI: D Hogan (Dartmouth)

\item [2017-2028] National Institutes of Health. R01-AI130128 \\
  ``Evolution of \textit{Aspergillus fumigatus} virulence'' \\
Role: Senior Personnel. PI: RA Cramer, Jr (Dartmouth)

\item [2019-2025] Canadian Institute For Advanced Research. Fellowship \\
  ``Fungal Kingdom: Threats and Opportunities'' \\
Role: CIFAR Fellow. PI/Directors: L Cowen and J Heitman

\item [2020-2023] Gordon and Betty Moore Foundation \\
  ``New Tools for Advancing Model Systems in Aquatic Symbiosis''  \\
Role: Co-PI. PI: Lillian Fritz-Laylin (U Mass-Amherst). With Co-PI Tim James (U Michigan)

\item [2020-2025] USDA-NIFA, Emergency Citrus Disease Research and Extension \\
``CAP: Combining Cultural And Genetic Approaches For Grove Success To Unravel And Enhance Resistance/Tolerance To Huanglongbing.'' \\
Role: Co-PI. PI Caroline Roper, UCR

\item [2022-2024] Canadian Institute For Advanced Research. Catalyst Award \\
  ``Discovering and describing fungi from deep biosphere environments'' \\
Role: PI. Co-PI: Tim James, U Michigan

\item [2022-2026] National Science Foundation. EF-2125066. \\
``Collaborative Research: MIM: Gut-inhabiting fungi influence structure and function of herptile microbiomes through horizontal gene transfer and novel metabolic function'' \\
  Role: PI. Collaborative linked award with 3 other PIs: J Spatafora \& K McPhail (Oregon State), D Walker (Middle Tennessee State)
\url{https://herptilemicrobiomes.org/}

\item [2022-2026] National Science Foundation. IOS-2134912 \\
  ``Research-PGR: Impact of transposable element bursts on the rice genome and epigenome.'' \\
Role: Co-I. PI: SR Wessler (UC Riverside). Co-I: R Schmitz (U Georgia), K Ostivek (UC Riverside), J Burnette (UC Riverside)

\item [2022-2026] National Science Foundation. DBI-2215705 \\
``Research Infrastructure: MRI: Acquisition of a Big Data HPC Cluster for Interdisciplinary Research and Training.'' \\
Role: Co-I. PI: Thomas Girke (UC Riverside). Co-I: Wenxiu Ma, Mark Alber, Adam Godzik (UC Riverside)

\item [2022-2027] National Science Foundation. IOS-2141858 \\
``CAREER: Dissecting the molecular regulation of septin-mediated plant invasion by the blast fungus \textit{Magnaporthe oryzae}'' \\
Role: Senior Personnel. PI: Martin Egan (U Arkansas)

\item [2024-2025] California Department of Food and Agriculture \\
``Blocking the Acquisition and Transmission Cycle of \textit{Xylella fastidiosa} by Glassy-winged Sharpshooter using Genetic Control.''
Role: Co-PI. PI: Peter Atkinson (UCR)

\item [2024-2025] USDA-APHIS \\
``CRISPR-mediated Genome Modification of \textit{Homalodisca vitripennis} for the genetic control of Pierce’s Disease.''
Role: Co-PI. PI: Rick Redak (UCR)

\item [2023-2026] California Department of Food and Agriculture - Specialty Crop Block Grant \\
``Rapid-response detection and management of emerging outbreaks of lettuce Fusarium wilt in Coastal California.''
Role: Co-PI. PI: Alexander Putnam (UCR)

\item [2023-2024] California Department of Food and Agriculture \\
``Eliminating transmission of \textit{Xylella fastidiosa} by the glassy-winged sharpshooter (\textit{Homalodisca vitripennis}) using CRISPR technology.''
Role: Co-PI. PI: Peter Atkinson (UCR)

\item [2024-2030] National Science Foundation. DBI-2400327 \\
``BioFoundry: A BioFoundry for Extreme \& Exceptional Fungi, Archaea and Bacteria (Ex-FAB)'' \\
Role: Co-PI. PI: Michelle O'Malley (UCSB)

\item [{\bf Completed support}]

  % \$500k \\
\item [2010-2013] Burroughs Wellcome Fund. \\
 ``FungiDB: A Pan Fungal Genome Database''. \\
Role: Co-I. PI: DS Roos (U Pennsylvania)

% $25K
\item [2011-2012] UC Riverside, Chancellor's Strategic Investment
  Funds. \\
``Coelomomyces Genomics for Mosquito Vector Control'' \\
Role: Co-I. PI: B Federici. Co-I: A Ray (UC Riverside)

%  \$50k \\
\item [2013-2014] UC Riverside, Office of Research Strategic Investment
  Funds. \\
``High-throughput synthetic biology for natural products discovery'' \\
Role: Co-I. PI: K Borkovich. Co-I: C Larive (UC Riverside)

% \$168k \\
\item [2013-2014] National Institutes of Health - 1-R03-AI105636-01. \\
``Annotation of \textit{Cryptococcus} genomes by comprehensive
  curation of published literature'' \\
Role: PI. Co-I G Sherlock (Stanford)

% \$750k \\
  \item [2011-2014] Alfred P. Sloan Foundation. \\
``MoBe DAC: A data coordinating center for the Sloan Indoor
Environment Metagenomic Project - Fungal resources''. \\
Role: PI.  Linked grants with F Meyer (U Chicago/ANL), R Knight (U Colorado), M Sogin (Marine Biological Lab).

% \$548k \\
\item [2014-2015] National Science Foundation. DBI-1429826. \\
``MRI: Acquisition of a Big Data Compute Cluster for Interdisciplinary
  Research''
Role: Co PI. PI T Girke. Co-Is J Bailey-Serres, M Allen, and S Lonardi (UCR)

\item [2014-2017] National Institutes of Health - 1-R01-GM108492-01. \\
``Dynamics of bacterial-fungal interactions in chronic lung infections'' \\
  Role: Co-I. PI: D Hogan (Dartmouth)

  \item [2011-2016] W.M. Keck Foundation. (No Cost Extension thru 2018) \\
''New Active Transposable Elements for Mosquito Genetics.'' \\
Role: Co-I. PI: SR Wessler (UC Riverside). Co-I: P Atkinson (UC Riverside).

\item [2017] Burroughs Wellcome Fund. \\
''Meeting grant to support Fungal Cell Wall (FCW2017) Conference in Ensenada, Mexico'' \\
Role: PI.

% \$317k (\$709k total award) \\
\item [2016-2019] National Science Foundation. DEB-1557110. (No Cost Ext thru 04/2020) \\
``Collaborative Research: Phylogenomics and evolutionary history of the anaerobic fungal group, Neocallimastigomycota'' \\
Role: PI. Collaborative linked award PI: N Youssef (Oklahoma State)

% 4.9M
\item [2011-2017] National Science Foundation. IOS-1027542.  (No Cost Ext thru 02/2021) \\
  ``CPGS: Genome-wide impact of \textit{mPing} transposition on rice phenotypic diversity.'' \\
Role: Co-I. PI: SR Wessler (UC Riverside). \\
\url{http://dynamiterice.org}

% 714k \$714k (\$2.5M total award) \\
\item [2015-2018] National Science Foundation. GO Life DEB-1441715. (No Cost Ext thru 08/2020) \\
``Collaborative Research: The Zygomycetes Genealogy of Life
  (ZyGoLife)- the conundrum of Kingdom Fungi'' \\
  Role: PI. Collaborative linked award with 3 other PIs and ~12
  collaborating labs: J Spatafora (Oregon State), TY James (U
  Michigan), R Robertson (Arizona State) \\
\url{http://zygolife.org}

% \$105k (\$1.7M total award) \\
\item [2017-2020] Univ of California-Office of the President, MRPI. \\
``UC Valley Fever Research Initiative'' \\
Role: Co-PI.  PI: Anita Sil (UCSF) and Co-PIs at UC Berkeley, UC Merced,
UC San Diego

\item [2019-2020] City of Hope / Univ of California-Riverside \\
``Antifungal drug resistance in Southern California: Discovery of
  novel mechanisms by genomics and proteomics.'' \\
Role: PI with Co-PIs M Kalkum and S Dadwal at City of Hope Hospital

\item [2020-2021] Canadian Institute For Advanced Research \\
  ``Pilot investigation of avian-origin \textit{Aspergillus fumigatus} infections
in the United States'' \\
Role: PI. Co-PI: David Blehert, National Wildlife Health Center, USGS

\item [2020] Burroughs Wellcome Fund. \\
''Meeting grant to support 2022 Fungal Cellular and Molecular Biology Gordon Research Conference'' \\
Role: PI.

\item [2020-2021] USDA-APHIS \\
``Tracking seasonal changes of endophytic communities in \textit{Fusarium} dieback - Invasive shot hole borers host trees in California.'' \\
Role: Co-I. PI Akif Eskalen, UC Davis

\item [2022] National Science Foundation. MCB-2227426 \\
''Meeting grant to support Fungal Cellular and Molecular Biology Gordon Research Conference 2022'' \\
Role: PI.

\item [2019-2022] Univ of California-Office of the President \\
``Investigating fundamental gaps in Valley Fever research'' \\
Role: Co-PI.  PI: Anita Sil (UCSF) and Co-PIs at UC Berkeley, UC
Davis, UC Merced, UC San Diego

\item [2019-2022] National Institutes of Health. R15-GM132869 \\
  ``Understanding The Mechanisms Of Spatial Protein Quality Control In A Model Filamentous Fungus'' \\
Role: Senior Personnel. PI: Egans, M (U Arkansas)

\item [2020-2022] California Conservation Genomics Project (subproject) \\
  ``Landscape and Population Genomics of the lichen \textit{Acarospora socialis} in California''  \\
Role: PI.

\item [2021-2023] Canadian Institute For Advanced Research. Catalyst Award \\
  ``Exploring the extended phenotypes of BdDV-1, a DNA mycovirus associated with enzootic strains of amphibian chytridiomycosis'' \\
Role: Co-PI. PI: Tim James, U Michigan; Co-PI: Lillian Fritz-Laylin, U Mass Amherst; Co-PI Mat Fisher, Imperial College (UK)

\item [2020-2023] California Department of Agriculture / Glassywinged Sharptshooter Board  \\
``CRISPR-mediated genome modification of \textit{Homalodisca vitripennis} for the genetic control of Pierce's disease'' \\
Role: Co-PI. PI Peter Atkinson, UCR

\end{cvlistcompact}

\setlength{\cvlabelwidth}{18mm}

\begin{cvlistcompact}{\bf Service:}
\item[{\bf University and Departmental}]
\item [2023-2024] Vice Chair, Department of Microbiology and Plant Pathology
\item [2023-2024]  Chair, ad hoc Senate commitee for evaluation of BS+MS programs
\item [2020-2022] Division Chair, Riverside Division of the University of California Academic Senate
\item [2020-2022] Member UC Academic Senate Academic Council (as per role as UCR Senate Chair)
\item [2021-2022] Member Senate/UCOP Leadership Budget Call (as per role as UCR Senate Chair)
\item [2020-2021] Member UC Academic Planning Committee (as per role as UCR Senate Chair)
\item [2020-2021] Member UCR Campus Safety Taskforce (as per role as UCR Senate Chair)
\item [2021] Member UCR Provost Search Committee (as per role as UCR Senate Chair)
\item [2018--2020] Chair, UC Riverside Graduate Council and member of Senate Executive Council
\item [2017-2018] Member, UC Riverside Graduate Council
\item [2015--2020] Director, Microbiology Graduate Program (except Sabbatical 2016-17)
\item [2014--2015, 2018--2020] Graduate Advisor, Microbiology Graduate Program
\item [2015--2016,2017--2018] Admissions Advisor, Microbiology Graduate Program
\\

\item[{\bf Editorial Boards}]
\item [2021--] Editorial Board, Annual Reviews of Microbiology
\item [2019--2023] Associate Editor, Genome Biology \& Evolution
\item [2019--2024] Associate Editor, Mycologia
\item [2018--] Senior Editor, Microbial Resource Announcements
\item [2018--] Associate Editor, GENETICS
\item [2016--] Editorial Board, Current Opinion in Microbiology
\item [2015--2019] Associate Editor, Microbial Genomics
\item [2014--2022] Associate Editor, Fungal Genetics \& Biology
\item [2013,2015] Guest Associate Editor, PLoS Genetics
\item [2013] Guest Associate Editor, Mycologia
\item [2011--2016] Faculty Member in Microbial Genetics \& Genomics, Faculty of 1000
\item [2010--2015] Editorial Board, Eukaryotic Cell.
\item [2009--2016] Section Editor, PLoS One.
\item [2007--2016] Academic Editor, PLoS One.
\\

\item [{\bf Professional Service}]
%\item [2023--] Consultant, Michroma.
%\item [2021--] Scientific Advisory Board, Sincarne.
\item [2018--2022] Co-Chair (2020, moved to 2022) of Cellular and Molecular Fungal Biology, Gordon Research Conference; Co-Vice Chair (2018).
\item [2017--2020] Karling Lecture Committee, Mycologia Society of America (Chair 2019-2020)
\item [2018-2021] Councilor for Cell Biology \& Physiology. Mycological Society of America.
\item [2014--2018] Neurospora Policy Committee, Co-Organized 2016 Neurospora conference
\item [2013--2019] Fungal Genetics Policy Committee
\item [2012--2020] Scientific advisory board, Plant Microbe Interactions - DOE Science Focus Area, Oak Ridge National Laboratory
\item [2012--2018] Scientific advisory board, WormBase
\item [2012--2015] Scientific advisory board, EnsEMBL Genomes
\item [2010--2012] Councilor for Genetics \& Molecular Biology, Mycological Society of America
\item [2009--2010] Advisory Board for Genomic Encyclopedia of Fungi, Joint Genome Institute, US Department of Energy.
\item [2009--2010] Pan-Fungal Database Steering Committee for Burroughs Welcome Fund.
\item [2007--2009] Scientific advisory board NSF Computer Science Education Revitalization (PI Owen Astrachan, Duke University)
\item [2005--2008] Scientific advisory committee Information Technology and
  Computing infrastructure, National Center for Evolutionary Synthesis (NESCent).
\item [2005--2011] President and Board Member [2005--2014], Open Bioinformatics Foundation \url{http://www.open-bio.org/}
\item [2001--2015] Co-Project leader, BioPerl. \url{http://www.bioperl.org/}
%\item [2002-2008] Program Committee, Bioinformatics Open Source Conference; Chair (2002).
\end{cvlistcompact}

%\begin{cvlistcompact}{Membership in Professional Societies:}
%\item [2015--] American Association for the Advancement of Science
%\item [2007--] Mycological Society of America
%\item [2007--] American Society for Microbiology, Fellow (2020)
%\item [2004--] Genetics Society of America
%\item [2004--] Society for Molecular Biology and Evolution
%\item [2002--] Open Bioinformatics Foundation
%\item [2002--2018] International Society for Computational Biology
%\end{cvlistcompact}

%\begin{cvlistcompact}{Departmental Service:}
%\item [2010-2011] Admissions committee, Microbiology graduate program
%\item [2010-2011] Seminar committee, Plant Pathology \& Microbiology
%\item [2010-2011] Website committee, Microbiology Graduate program
%\item [2010-2011] Curriculum commitee, Genetic, Genomics, \&
%  Bioinformatics program
%\item [2010-2011] Social affairs committee, Plant Pathology \& Microbiology
%\item [2009-2010] Space and Infrastructure committee, Plant Pathology \& Microbiology
%\end{cvlistcompact}

\begin{cvlistcompact}{Graduate Students:}
\item [2009--2013] PhD student, Divya Sain. Genetics, Genomics, \&
  Bioinformatics. \\ Current: Scientific Program Manager at Velsera.
\item [2010--2012] MS student, Yi (Zoe) Zhou. Genetics, Genomics, \&
  Bioinformatics. \\ Current: Biostatistician at dMed Biopharmaceutical Co.
\item [2010--2014] PhD student, Yizhou Wang. Plant Biology. \\ Current:
  Research Bioinformatician III in Data Science Navigator Team, Cedars-Sinai.
\item [2011--2015] PhD student, Steven Ahrendt. Genetics, Genomics, \&
  Bioinformatics. \\ Current: Data Scientist at DOE Joint Genome Institute.
\item [2016--2019] PhD Student, Derreck Carter-House. Plant Pathology. \\ Current: Senior Scientist, Clear Labs
\item [2015--2021] MS Student, Sawyer Masonjones. Genetics, Genomics, \&
  Bioinformatics
\item [2015--2021] PhD Student, Nuttapon Pombubpa. Plant Pathology. \\ Current: Assistant Professor, Chulalongkorn, Bangkok, THAILAND
\item [2016--2022] PhD Student, Jes\'{u}s Pe\~{n}a, Microbiology. \\ Current: Visiting Assistant Professor, Colorado College
\item [2017--2022] PhD Student, Tania Kurbessoian, Microbiology. \\ Current: Postdoctoral Fellow, University of North Carolina at Chapel Hill
\item [2017--2024] PhD Student, Julia Adams, Plant Biology
\item [2020--2024] PhD Student, Talieh Ostovar, Evolutionary Biology, San Diego State - UCR Joint Doctoral Program
\item [2021--2024] PhD Student, Mark Yacoub, Microbiology
\item [2021--] PhD Student, Cheng-Hung Tsai, Genetics, Genomics, \& Bioinformatics
\item [2022--] PhD Student, Jessica Wu-Woods, Microbiology
\item [2022--] PhD Student, Leila Shadmani, Microbiology
\item [2022--] PhD Student, Xueyan (Sharon) Xu, Cellular, Molecular, and Developmental Biology.
\item [2023--] PhD Student, Kian Kelly, Plant Pathology
\item [2023--] PhD Student, Nathan Matheiu, Genetics, Genomics, \& Bioinformatics
\item [2023--] PhD Student, Nora Ismail, Microbiology
\item [2024--] PhD Student, Julissa Perez-Maron, Microbiology

\end{cvlistcompact}

\begin{cvlistcompact}{Postdoctoral Fellows:}
\item [2010--2011] John Abramyan, Ph.D. \\
  Current: Associate Professor, Univ of Michigan-Dearborn
\item [2011--2014] Sofia Robb, Ph.D. \\
  Current: Genomics Scientist at Stowers Institute.
\item [2012--2014] Brad Cavinder, Ph.D. \\
  Current: Research Associate at Michigan State University
\item [2012--2015] Peng Liu, Ph.D. \\
  Current: Research Associate, Yangzhou University, CHINA
\item [2013--2019] Jinfeng Chen, Ph.D. \\
Current: Assistant Professor, Institute of Zoology of Chinese Academy of Science; 1st position: Staff Scientist, City of Hope, CA.
\item [2013--2015] Ousmane Ciss\'{e}, Ph.D. - Swiss National Science Foundation Fellow. \\
  Current: Staff Scientist at Critical Care Department, NIH Clinical Center.
\item [2014--2015] Rodrigo Olarte, Ph.D. \\
  Current: Postdoctoral Fellow at Univ of Minnesota.
\item [2017--19] Yan Wang, Ph.D. \\
  Current: Assistant Professor, University of Toronto-Scarbourgh.
\item [2019--2021] Lotus Lofgren, Ph.D. \\
  Current: Incoming Assist Professor, University of California-Berkeley \\
  Previous: Tri-I Mycology Postdoctoral Researcher, Duke University.
\item [2020--2021] Ying Sun, Ph.D. \\
Current: Postdoctoral Researcher at Salk Institute
\item [2020--] Cassie Ettinger, Ph.D.
\item [2020--2023] Kelsey Aadland, Ph.D.  \\
Current: Computational Biologist at Computercraft Corporation, NCBI Protein Domains team
\item [2023--2024] Claudia Coleine, Ph.D. - Marie Curie Fellow.
\item [2023--] Carolina Pina-Paez, Ph.D.
\end{cvlistcompact}

\begin{cvlistcompact}{Visitors:}
\item [2010--2013 (4, 2-3 month vists)] Anastasia Gioti, PhD,
  Dept of Evolution Biology, Uppsala University, SWEDEN
\item [2010] Suzanne Joneson, PhD, Department of Biology, University of Idaho
\item [2011] Edgar Medina Tovar, MSc
  Mycology and Phytopathology Lab, Universidad de Los Andes, Bogota, COLOMBIA
\item [2012] Andrii Gryganski, PhD, Visiting Researcher, Duke University
\item [2013--2014] Venkatesh Moktali, PhD,
  FungiDB Project, Visiting Research Fellow, Oregon State University
\item [2014] Ra\'{u}l Castanera Andr\'{e}s, Visiting Graduate Student,
  Universidad P\'{u}blica de Navarra, Pamplona, SPAIN
\item [2015] Natalie Vande Pol, Visiting Graduate Student (Bonito Lab), Michigan State University
\item [2015--2016] Zhinquan Song, Visiting Graduate Student (Guangyi
  Wang Lab), Tianjin University, CHINA
\item [2015] John Yinka Odebode, Visiting Graduate Student on a West
  African Research Assocation Fellowship, University of Lagos, NIGERIA.
\item [2015] Marco Marconi, Visiting Graduate Student,
    Universidad Polit\'{e}cnica de Madrid, Madrid, SPAIN
\item [2015--2016] Claudia Coleine, Visiting Graduate Student,
    Universit\'{a} degli Studi della Tuscia, Viterbo, ITALY
\item [2017] Jane Lind Nybo, Visiting Graduate Student,
Technical University of Denmark, Copenhagen, DENMARK
\item [2019] Guillermo Vidal-Diez de Ulzurrun, Visiting Postdoc scientist,
IMB, Academia Sinica, Taipei, Taiwan
\item [2019--2020] Felipe Salgado, Federal University of Rio de Janeiro, BRAZIL.
\item [2020--2021] Omar Valencia, Volunteer.
\item [2021--2022] Jaehyuk Choi, Incheon National University, SOUTH KOREA.
\item [2022--2023] Xinzhan Liu, Institute of Microbiology, Chinese Academy of Sciences, CHINA.
\end{cvlistcompact}

\begin{cvlistcompact}{Staff:}
\item [2011--2012] Daniel Borcherding, Programmer (FungiDB). \\ Current: Senior Software Build Engineer, Apple, Inc.
\item [2011--2013] Raghuraman Ramamurthy, Programmer (FungiDB). \\ Current: Lead Bioinformatician - Natera.
\item [2012--2014] Edward Liaw, Programmer (FungiDB). \\ Current: Bioinformatics Engineer - Twist Bioscience.
\item [2012--2014] Greg Gu, Programm (FungiDB). \\ Current: Chief Engineer - PH Engineering Corp.
\item [2013--2014] Venkatesh Moktali, Bioinformatics Scientist (FungiDB). \\ Current: Biotech and Healthcare Product Management -  Twist Bioscience.
\item [2017--2018] Jericho Ortanez, Junior Specialist. Current: Graduate Student, UC Riverside.
\item [2021] Omar Valencia, Junior Specialist.
\item [2022--2023] Sadikshya Sharma, Assistant Specialist.
\end{cvlistcompact}

\begin{cvlistcompact}{Teaching:}
\item [2010,2012] BIO5C - Introductory Ecology \& Evolution
\item [2011] BIO20 - The Dynamic Genome - Research module for \textit{Neurospora} research
\item [2011,2013] GEN240B - Tools for Bioinformatics and Genome Analysis
\item [2015] MCBL124 - Microbial Pathogenesis
\item [2011--2016] MCBL211 - Microbial Ecology
\item [2012-2015] MCBL202 - Microbial Pathogenesis \& Physiology
\item [2012--Present] GEN220 - Computational Analysis of High Throughput Biological Data \url{http://biodataprog.github.io/}
\item [2016--2020] BIO119 - Introduction to Genomics and Bioinformatics
\item [2022--Present] MCBL221 - Microbial Genetics
%\item [2024--Present] MCBL125 - Experimental Microbiology

\end{cvlistcompact}

\begin{cvlistcompact}{Undergraduate Researchers:}
\item[2010--] Sponsor for summer research students in MARCU, STEM, and CAMP
  programs at UCR.
\item [2010--2012] Jessica De Anda, UCR. STEM grant participant
  (2010); MARC USTAR student 2010-12. Current: Career Development Coordinator at UC Berkeley School of Buisiness
\item [2010--2011] Annie Nguyen, UCR.
\item [2011--2012] Carlos Rojas Torres, UCR. CAMP (2011); lab
  researcher. Current: Gilead Pharmaceuticals.
\item [2011] Ramy Wissa, UCR. Pre-MARC USTAR Summer student.
\item [2011--2012] Lorena Rivera, UCR. Pre-MARC USTAR student (2011); lab
  researcher, CNAS Dean's Fellow Summer Undergraduate Research (Summer 2012)
\item [2012--2014] Erum Khan, UCR.
\item [2012--2014] Sapphire Ear, UCR. Current: MD student at UCSF
\item [2012--2014] Megna Tiwari, UCR. Current: PhD student at Univ of Georgia
\item [2013--2014] Dylan McVay, UCR.
\item [2013--2016] Na Jeong, UCR, Summer RISE Scholar (2013) and lab researcher
\item [2014] Spencer Swansen, Summer NSF REU student (Seattle Pacific University)
\item [2015--2017] Justin Shen, UCR.
\item [2015--2016] Serena Choi, UCR.
\item [2015--2017] Dillon McDonald, UCR Summer HSI-STEM (2015) and lab researcher. Current: DO Student, Western University of Health Sciences in Oregon
\item [2015] Christina Uriarte, UCR. Pre-MARC USTAR student.
\item [2015--2017] Jericho Ortanez, UCR. Current: PhD student UCR Microbiology
\item [2015--2016] Leandra Ibrahim, UCR.
\item [2015--2017] Deane Kim, UCR.
\item [2016--2017] Georgiy Smirnov, UCR.
\item [2016--2018] Meng (Josh) Chung, UCR. Current: Dentistry Student
\item [2017--2019] Estefania Caldera, UCR.
\item [2018] Lily Bautista, UCR.
\item [2018--2020] Renata Haro, UCR.
\item [2018--2020] Skylar McDonald, UCR. Current: MS student in UCR Engineering.
\item [2019] Saisuki Putumbaka, The College of New Jersey, Summer REU student. Current: PhD student at Univ of Georgia
\item [2019--2020] Nicole Leung, UCR.
\item [2020--2021] Dionne Martin, UCR - won IIGB Undergraduate Research Award. Next: PhD student at Univ of Georgia
\item [2021--2023] Amy Do, UCR. Current: MS student in UCR Engineering.
\item [2023] Jared Coyle, CSUSB. - Summer REU student.
\item [2023--] Varshini Balaji, UCR.
\item [2023--] Mia Miyatake, UCR.


\end{cvlistcompact}

\begin{cvlistcompact}{Thesis/Dissertation committees:}
\item [2011] Sourav Roy, PhD, Genetics, Genomics \& Bioinformatics
\item Yi Zhou, MS, Genetics, Genomics \& Bioinformatics $\star$
\item [2012] Andrew Defries, PhD, Plant Sciences
\item [2013] Gilbert Uribe, MS, Plant Pathology
\item Divya Sain, PhD, Genetics, Genomics \& Bioinformatics $\star$
\item [2014] Yizhou Wang, PhD, Plant Sciences $\star$
\item Zhigang Wu, PhD, Genetics, Genomics \& Bioinformatics
\item [2015] Presha Shah, PhD, Biochemistry
\item Ming Wang, PhD, Plant Pathology
\item Steven Ahrendt, PhD, Genetics, Genomics \& Bioinformatics $\star$
\item Ilva Cabrera, PhD, Genetics, Genomics \& Bioinformatics
\item Jinfeng Lu, PhD, Genetics, Genomics \& Bioinformatics
\item James Ricci, MS, Entomology
\item [2016] Ryan Arvidson, PhD, Biochemistry
\item Francis Na, MS, Microbiology
\item Jishu Ha, PhD, Genetics, Genomics \& Bioinformatics
\item Arit Gosh, PhD, Genetics, Genomics \& Bioinformatics
\item Kelsey Gano, PhD, Microbiology
\item Kun Liu, PhD, Plant Biology
\item [2017] Raissa Green, PhD, Genetics, Genomics \& Bioinformatics
\item Amelia Lindsey, PhD, Entomology
\item Patrick Schriener, PhD, Genetics, Genomics \& Bioinformatics
\item Eric Smith, PhD, Genetics, Genomics \& Bioinformatics
\item Katherine Picard, PhD, Univ Prog in Genetics \& Genomics (Duke University)
\item Eric Gordon, PhD, Entomology
\item [2018] Cynthia Dick, PhD, EEOB
\item Dan Vanderpool, PhD, Biology (University of Montana)
\item Steven Bolaris, PhD, Genetics, Genomics \& Bioinformatics $\triangle$
\item [2019] Joseph Carrillo, PhD, Plant Pathology $\triangle$
\item Dinusha Maheepala Mudalige, PhD, Plant Biology
\item Aaron Robinson, PhD, Biology (University of New Mexico)
\item Courtney Collins, PhD, Plant Biology
\item Edgar Medina, PhD, Univ Prog in Genetics \& Genomics (Duke University)
\item Lluvia Vargas, PhD, Microbiolog\'{i}a (CICESE, MEXICO)
\item Derreck Carter-House, PhD, Plant Pathology $\star$
% \item Nathan Robinett, Evolutionary Biology, Joint Doctoral Prog. SDSU-UCR $\circ$
\item [2020] Andrea Vu, PhD, Plant Pathology
\item Nichole Ginnan, PhD, Plant Pathology
\item Alex Rajewski, PhD, Plant Biology
\item[2021] Nuttapon Pombubpa, PhD, Plant Pathology $\star$
\item Caleb Hubbard, PhD, Medical and Veterinary Entomology
\item Sawyer Masonjones, MS, Genetics, Genomics \& Bioinformatics $\star$
\item Markus Hiltunen, PhD, Evolutionary Biology, Uppsala University (external opponent)
\item [2022] Yi Huang, PhD, Plant Biology
\item Jes\'{u}s Pe\~{n}a, PhD, Microbiology $\star$
\item Hannah Shulman, PhD, Microbiology
\item Christopher Fiscus, PhD, Genetics, Genomics \& Bioinformatics
\item Celia Xi, PhD, Plant Biology
\item Sarah Thorwall, Chemical and Environmental Engineering
\item Tania Kurbessoian, Microbiology $\star$
\item Moira Kelly, Ghent University (external PhD Exam committee)
\item [2023] Robyn Anderson, University of Western Australia (external PhD Exam committee)
\item Samantha (Smith) Standring, Entomology
\item Zachary Konkel, The Ohio State University (external PhD Exam committee)
\item Glen Morrison, Plant Biology
\item Peggy Brady, EEOB
\item Danielle Stevenson, Environmental Sciences
\item Fabiola Pulido-Chavez, Plant Pathology
\item Aidan Shands, Plant Pathology
\item [ongoing] Julia Adams, Plant Biology $\star$
\item Talieh Ostovar, Program in Evolutionary Biology SDSU-UCR $\star$
\item Dylan Enright, Microbiology
\item Yagna Oza, Genetics, Genomics, Bioinformatics
\item Jericho Orta\~{n}ez, Microbiology
\item Linton Freund, Genetics, Genomics, \& Bioinformatics
\item Isaac Diaz, Genetics, Genomics, \& Bioinformatics
\item Tamsen Dunn, Program in Evolutionary Biology SDSU-UCR
\item Angela Buehlman, Plant Biology
\item Colin Todd, Plant Biology
\item Ben Hoyt, Plant Pathology
\item Mark Yacoub, Microbiology $\star$
\item Jessica Maccaro, Entomology
\item Aida Tafrishi, Chemical and Environmental Engineering
\item Jessica Wu-Woods, Microbiology $\star$
\item Leila Shadmani, Microbiology $\star$
\item Xueyan (Sharon) Xu, Cellular, Molecular and Developmental Biology $\star$
\item Nathan Mathieu, Genetics, Genomics, Bioinformatics $\star$ (co-advised with Sue Wessler)
\item Sadikshya Sharma, Genetics, Genomics, Bioinformatics $\star$
\item Kian Kelly, Plant Pathology $\star$
\item Nora Ismail, Microbiology $\star$
\item Julissa Perez-Maron, Microbiology $\star$

\end{cvlistcompact}
$\star$ Stajich is Dissertation advisor or $\triangle$ co-advisor / substitute
%$\circ$ Withdrawn from program
% cvlist items can take dates as a parameter.

\setlength{\cvlabelwidth}{8mm}
\begin{cvlistcompact}{\bf Invited Seminars and conference presentations (2015--Present)}
\item [2024] $\cdot$ Louisiana State University, Graduate Student Invited Speaker, Dept of Plant Pathology
\item $\cdot$ 29th Fungal Genetics Conference. Pacific Grove, CA.
\item [2023] $\cdot$ University of Arizona, Guest Lecturer in Mycology Class (Virtual)
\item $\cdot$ FUN-EX - IUBMB Focused Meeting on Extremophilic Fungi, Llubjana, SLOVENIA
\item $\cdot$ University of Georgia, Dept of Microbiology
\item [2022] $\cdot$ CIFAR Fungal Kingdom: Threats \& Opportunities, Presenter for Feb and March Meetings (Virtual)
\item $\cdot$ Keynote speaker, Bark Beetle Mycobiome Research community meeting (Virtual)
\item $\cdot$ Scripps Institution of Oceanography, UC San Diegoi (Virtual)
\item $\cdot$ Mycological Society of Japan Annual Meeting (Virtual)
\item $\cdot$ National Academies Symposium on Valley Fever, Irvine, CA (Virtual presentation)
\item [2021] $\cdot$ CIFAR Fungal Kingdom: Threats \& Opportunities, Presenter for Feb and March Meetings
\item $\cdot$ University of Georgia, Guest lecture for undergraduate seminar course "Genome Biology Across the Tree of Life" (Virtual)
\item $\cdot$ Rochester Institute of Technology, Georgia Gosnell Seminar Series (Virtual)
\item $\cdot$ University of Deleware, Microbiology Graduate Program (Virtual)
\item $\cdot$ Canadian Fungal Network Conference, Plenary Speaker (Virtual)
\item $\cdot$ Botany / Mycological Society of America 2021 meeting (Virtual)
\item $\cdot$ Metaorganisms: Collaborative Research Center Seminar series, Germany (Virtual)
\item [2020] $\cdot$ Microbiology and Infectious Disease Grad Student retreat speaker, Univ Texas Health Sciences, Houston, TX (postponed)
\item [2019] $\cdot$ Phylogenomics Workshop, Cesky Krumlov, Czech Republic
\item $\cdot$ Middle Tennessee State University, Murfreesboro, TN
\item $\cdot$ Rosie Perez Memorial Seminar, North Carolina State University, Raleigh, NC
\item $\cdot$ University of North Carolina, Chapel Hill, NC
\item $\cdot$ California State University, Northridge, CA
\item [2018] $\cdot$ UC Riverside Data Science Series. Riverside, CA
\item $\cdot$ University of Nebraska-Lincoln, Lincoln, NE
\item $\cdot$ Creighton University, Omaha, NE
\item $\cdot$ Marine Fungi Workshop. Marine Biological Lab, Woods Hole, MA.
\item $\cdot$ 11th International Mycological Congress. San Juan, Puerto Rico
\item $\cdot$ CIFAR workshop "Microbial Pathogens in the Fungal Kingdom". Toronto, Ontario, CANADA
\item [2017] $\cdot$ Oregon State University. Corvallis, OR
\item $\cdot$ 29th Fungal Genetics Conference. \textit{Plenary Speaker}. Pacific Grove, CA.
\item $\cdot$ Oomycete Molecular Genetics Network. \textit{Plenary Speaker}. Pacific Grove, CA
\item $\cdot$ Population Genomics of Oomycete and Fungal Pathogens. Ascona, Switzerland
\item $\cdot$ American Society for Microbiology Microbe Meeting. New Orleans, LA
\item $\cdot$ FASEB Microbial Pathogenesis. Aspen, CO.
\item $\cdot$ Mycological Society of America 2017 Meeting. Athens, GA
\item $\cdot$ American Academy of Microbiology Colloquium on Fungal
  Pathogenesis. Washington, DC
\item $\cdot$ Fungal Cell Wall Conference. Ensenada, Mexico
\item $\cdot$ Whetzel-Westcott-Dimock Special Lecturer, Cornell University, Ithaca, NY
%\item [2016] $\cdot$ Mycological Society of America 2016 Meeting. Berkeley, CA.
%\item $\cdot$ CIFAR Integrated Microbial Biodiversity Program. Toronto, ON, CANADA.
%  \item $\cdot$ 13th European Fungal Genetics Conference. Paris, France.  {\it Plenary Speaker}
%  \item $\cdot$ Neurospora Conference. Asilomar Conference Center, Pacific
%    Grove, CA.
%  \item $\cdot$ Duke University, Durham, NC.
%  \item $\cdot$ University of California, Davis, CA.
%  \item $\cdot$ University of Exeter. United Kingdom.
%\item [2015] $\cdot$  EMBO Conference:  Genomic complexity and diversity of eukaryotes. Sant Feliu de Guixols, SPAIN.  % \textit{Evolutionary Genomics of Early Branches  of the Fungal Tree of Life} \\
%\item $\cdot$  XI CONGRESO NATIONAL DE MICOLOGIA, Sociedad Mexicana de
%  Micologia. Merida, Yucatan, MEXICO. {\it Plenary Speaker}  %  \textit{Comparing molecular and  morphological fungal evolution through studies of fungal genomes} \\
%  \item $\cdot$ University of Arizona, Tucson, AZ. %  \textit{Evolution of complexity in fungi from evolutionary genomics of early diverging lineage} \\
%  \item $\cdot$  Eighth International Conference on Mycorrhiza. Flagstaff, AZ. % \textit{Evolutionary genomics of early branches of the   fungal tree \& the ZyGoLife project} \\
%  \item $\cdot$ %\textit{Evolutionary genomics of early branches
    %of the fungal tree: A view from the 1000 Fungal Genomes Project
   % and ZyGoLife Project} \\
%  Mycological Society of America 2015 Meeting. Edmonton, AB, CANADA.
%  \item $\cdot$  Society for Molecular Biology \& Evolution
%    2015. Vienna, Austria. %\textit{Evolutionary genomics of early
% branches of the fungal tree: The 1000 Fungal Genomes Project} \\
%  \item $\cdot$  University of California, Los Angeles, CA.
%    \textit{Evolutionary and functional genomics in early
%    diverging fungi}    Bioinformatics Seminar series. \\
%  \item $\cdot$  University of California, Merced, CA.
%     \textit{Evolutionary and functional genomics in early
%    diverging fungi} \\
%  \item $\cdot$  28th Fungal Genetics Conference. Asilomar Conference
%    Center,  Pacific Grove, CA.
%     \textit{Phylogenomics of the Zygomycete lineages: Exploring
%    phylogeny and genome evolution} \\
%  \item $\cdot$ Oregon State University, Corvallis, OR. % \textit{Evolutionary and functional genomics in early
%    %diverging fungi} \\
%  \item $\cdot$  Oklahoma State University, Stillwater, OK. \\

    %\textit{Clues to multicellular evolution of fungi from
    % comparative genomics \& studies of early diverging fungi} Dept of
    % Microbiology and Molecular Genetics. \\

%  \item [2014] $\cdot$ Stazione Zoologica Anton Dohrn. Naples,
%    ITALY. \textit{Evolutionary genomics and the origins of multicellular forms in filamentous fungi}
%
%\item [2011] $\cdot$ Center for Comparative Genomics \& Bioinformatics, Penn State University, State College, PA.
%\item $\cdot$ Department of Biology, San Diego State University, San Diego, CA
%\item $\cdot$ Bio-IT Cloud Summit, La Jolla, CA
%\item $\cdot$ Molecular Mycology Course, Marine Biological Lab, Woods Hole, MA
%\item $\cdot$ Mycological Society of America, Fairbanks, AK
%\item $\cdot$ Workshop in Comparative Genomics, Ft Collins, CO
%\item $\cdot$ $8^{th}$ Internation Cryptococcus and Cryptococcosis Conference, Charleston, SC
%\item $\cdot$ Symbiosis Workshop, Yosemite NP, CA
%\item $\cdot$ $26^{th}$ Fungal Genetics Conference, Asilomar Conference Center, Pacific Grove, CA. %(Session Chair: Comparative and Functional Genomics)
%\item $\cdot$ University of California-Davis, Department of
%  Plant Pathology, Davis, CA
%\item [2010] $\cdot$ National University of Ireland, Maynooth, Department of Biology, Maynooth, Ireland
%\item $\cdot$ Basidiomycete Genomics workshop, Joint Genome Institute User
%  meeting, Walnut Creek, CA
%\item $\cdot$ University of California-Riverside, GGB Seminar Series, Riverside, CA.
%\item $\cdot$ 2010 Neurospora Conference, Asilomar Conference Center, Pacific Grove, CA
%\item $\cdot$ PBoFF Symposium, Texas A \& M University, College Station, TX.
%\item $\cdot$ Cellular and Molecular Biology of Fungi, Gordon Research
%  Conference, Holderness, NH.
%\item $\cdot$ Mycological Society of America, Lexington, KY
%\item $\cdot$ $9^{th}$ International Mycological Congress, Scotland, UK
%\item [2009] $\cdot$ University of California-Riverside, Department of Computer Science, Riverside, CA.
%\item $\cdot$ University of California-Los Angeles, MSRI Bioinformatics Series,
%  Los Angeles, CA.
%\item $\cdot$ Comparative Genomics of Eukaryotic Microorganisms, EMBO
%  Conference, San Feliu de Guixols, Spain.
%\item $\cdot$ University of Idaho, Initative for Bioinformatics and
%  Evolutionary Studies, Moscow, ID.
%\item $\cdot$ Molecular Mycology Course, Marine Biological Lab,
%  Woods Hole, MA.
%\item $\cdot$ Mycological Society of America, Snowbird, UT.
%\item $\cdot$ Eccles Institute, University of Utah, Deptartment of Human Genetics, Salt Lake City, UT.
%\item $\cdot$ Bioinformatics Symposium \& Workshop, Oregon State University,
%  Corvallis, OR.
%\item $\cdot$ $25^{th}$ Fungal Genetics Conference, Asilomar Conference
%  Center, Pacific Grove, CA. %(Session Chair: Evolutionary Genetics \& Genomics)
%\item $\cdot$ University of Arizona, Genomics IGERT Program, Tucson, AZ.
%\item $\cdot$ Michigan State University, Pythium Genome Workshop, East
%  Lansing, MI.
%\item $\cdot$ Cornell University, Department of Plant Pathology, Ithaca, NY.
%\item $\cdot$ Graduate Research School in Genomic Ecology, Winter meeting,
%  Lund University, Lund, Sweden.
\end{cvlistcompact}

\end{cv}
\end{document}
